samples/highcharts/demo/stellar-chart/demo.js

```diff
@@ -0,0 +1,224 @@
+        }
+    }, {
+        type: 'pie',
+        dataLabels: {
+            enabled: false
+        },
+        size: '40%',
+        innerSize: '85%',
+        zIndex: -1,
+        point: {
+            events: {
+                mouseOver() {
+                    const {
+                            minDate,
+                            maxDate
+                        } = this.options,
+                        point = this,
+                        series = this.series,
+                        chart = series.chart,
+                        bubbleSeries = chart.series[0];
+
+                    bubbleSeries.points.forEach(point => {
+                        if (
+                            point.options.discoveryDate < minDate ||
+                            point.options.discoveryDate >= maxDate
+                        ) {
+                            point.graphic.attr({
+                                opacity: 0.2
+                            });
+                        }
+                    });
+
+                    series.customLabel = fillCenter(point.percentage,
+                        point.options.info, chart, series.customLabel);
+                },
+                mouseOut() {
+                    const chart = this.series.chart,
+                        series = this.series,
+                        bubbleSeries = chart.series[0];
+
+                    bubbleSeries.points.forEach(point => {
+                        point.graphic.attr({
+                            opacity: 1
+                        });
+                    });
+                    series.customLabel = fillCenter(100, '1990s-2020s', chart,
+                        series.customLabel);
+                }
+            }
+        },
+
+        data: [{
+            y: 12,
+            color: '#6CDDCA',
+            info: '1990s',
+            minDate: 1990,
+            maxDate: 2000
+        },
+        {
+            y: 47,
+            color: '#C771F3',
+            info: '2000s',
+            minDate: 2000,
+            maxDate: 2010
+        },
+        {
+            y: 117,
+            color: '#4D90DB',
+            info: '2010s',
+            minDate: 2010,
+            maxDate: 2020
+        },
+        {
+            y: 64,
+            color: '#FAB776',
+            info: '2020s',
+            minDate: 2020,
+            maxDate: 2030
+        }
+        ]
+    }]
+});
+
+function fillCenter(percentage, info, chart, customLabel) {
+    const labelText = `<span style='font-size: 2em'>☉</span><span
+    class='uppercase'><br><span class='decade'>${info}</span><br>
+    Planets discovered</span><br><span class='percentage'>
+    ${percentage.toFixed(2)}<sup style='font-size: 0.5em;'>%</sup></span>`;
+
+    if (!customLabel) {
+        customLabel = chart.renderer.label(labelText, 0, 0, void 0, void 0,
+            void 0, true
+        ).css({
+            color: '#000',
+            textAlign: 'center',
+            'pointer-events': 'none'
```

Comment:
For this to work it needs to be `pointerEvents` because of `useHTML` argument set to `true` in the constructor. Not sure if this is a bug. The style is set only for the empty group in SVG and has no effect on the HTML elements under the chart's SVG.

Reply:
I am not sure if that's a bug, we are mixing HTML and SVG elements, which overlaid with each other, so there is a good explanation for adding `pointEvents: none`.

Reply:
`'pointer-events': 'none'` doesn't work for me. Could you verify and try using  `pointerEvents: 'none'` instead?